src/collections/_documentation/server/config.md

```diff
@@ -57,14 +57,55 @@ Many settings available in `config.yml` will also be able to be configured in th
 
   A secret key used for session signing. If this becomes compromised it’s important to regenerate it as otherwise its much easier to hijack user sessions.
 
-  ```python
+  ```yaml
   system.secret-key: 'a-really-long-secret-value'
   ```
 
   To generate a new value, we’ve provided a helper:
 
   > $ sentry config generate-secret-key
 
+## Logging
+
+Sentry logs to two major places: `stdout`, and it's internal project. To disable logging to the internal project, add a logger who's only handler is `'console'` and disable propagating upwards.
+
+  {% include components/alert.html
+    content="The CLI flag and environment variable explained below override the root logger and anything in `LOGGING.overridable`. Be very careful with this in a production system, because the celery logger can be extremely verbose when set to INFO or DEBUG."
+    level="warning"
+  %}
+
+`-l/--loglevel`
+
+: Declared on the command line.
+
+  Sentry has the ability to override logger levels by providing the CLI with the `-l/--loglevel` flag.
+  The value of this can be one of the [standard python logging level strings](https://docs.python.org/2/library/logging.html#levels).
+
+  ```shell
+  sentry --loglevel=WARNING
+  ```
+
+`SENTRY_LOG_LEVEL`
+
+: Declared in system environment.
+
+  Sentry has the ability to override logger levels with the `SENTRY_LOG_LEVEL` environment variable.
+  The value of this can be one of the [standard python logging level strings](https://docs.python.org/2/library/logging.html#levels).
```

Comment:
```suggestion
  The value of this can be one of the [standard Python logging level strings](https://docs.python.org/2/library/logging.html#levels).
```